rotkehlchen/api/v1/resources.py

```diff
@@ -2398,3 +2403,39 @@ def post(
 class CounterpartiesResource(BaseMethodView):
     def get(self) -> Response:
         return self.rest_api.get_all_counterparties()
+
+
+class AddressbookResource(BaseMethodView):
+    get_delete_schema = AddressbookAddressesSchema()
+    update_schema = AddressbookUpdateSchema()
+
+    @use_kwargs(get_delete_schema, location='json_and_query_and_view_args')
```

Comment:
```suggestion
    @use_kwargs(get_delete_schema, location='json_and_view_args')
```

no need to enable query arguments

Reply:
Nope, we need them. Otherwise we get `"book_type": ["Missing data for required field."]}`

Reply:
Check my discord comments on why you need to change this.